src/linksys-addfwhdr.c

```diff
@@ -0,0 +1,191 @@
+/*
+	Linksys e8350 v1 firmware header generator
+*/
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/stat.h>
+#include <stdarg.h>
+#include <getopt.h>
+#include <time.h>
+#include <sys/time.h>
+#include "cyg_crc.h"
+#define AC2350          20
+
+#define CYBERTAN_VERSION        "v1.0.03"
+#define SERIAL_NUMBER   "003"
+#define MINOR_VERSION   ""
+#define BUILD_KEYWORD   " B"
+#define BUILD_NUMBER    SERIAL_NUMBER
+#define BETA_VERSION    " "
+#define CYBERTAN_UBOOT_VERSION      "v1.0"
```

Comment:
Add new line after include.

And maybe align every variable on the same colum with tabs ?

Reply:
Done